collect.py

```diff
@@ -128,6 +129,10 @@ def do_something(driver, elem_id=None):
             elem.click()
         elif input_type == 'number':
             elem.send_keys('3')
+        elif input_type == 'search':
+            elem.clear()
+            elem.send_keys('quick search')
+            elem.send_keys(Keys.RETURN)
```

Comment:
I'm not sure about pressing return here, let's only write the text for now.

Reply:
Okay.